### PR TITLE
chore: delete stale example dir

### PR DESCRIPTION
**Pull request by lucasgonze**
Signed-off-by: Lucas Gonze <lucas@gonze.com>

## Summary

Delete directory /magma/example

This seems to be unused. There are no substantive changes in years. I couldn't find any references to it in documentation. No developer on Slack recognized it.

## Test Plan

Push to draft PR, make sure CI tests pass that should pass. Apart from that, it's hard to know how to test this.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (ssanadhya) [inline lte/gateway/package-lock.json]: Does this file belong to this PR?

File: lte/gateway/package.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ssanadhya) [inline lte/gateway/package.json]: What's the purpose of adding this file in this PR?

**Review by ssanadhya — CHANGES_REQUESTED** [review]

File: lte/gateway/package-lock.json (hunk 1 above)
Comment (lucasgonze) [reply lte/gateway/package-lock.json]: Thank you for catching this anomaly. This change was intended for a different branch.

File: lte/gateway/package.json (hunk 2 above)
Comment (lucasgonze) [reply lte/gateway/package.json]: Thank you for catching this anomaly. This change was intended for a different branch.

**Review by ssanadhya — APPROVED** [review]
Lgtm

**Pull request merged**